src/overlays/gamestates/ovl_select/z_select.h

```diff
@@ -3,12 +3,39 @@
 
 #include "global.h"
 
+struct SelectContext;
+
 void Select_Init(GameState* thisx);
 void Select_Destroy(GameState* thisx);
 
+typedef void (*Select_LoadFunc)(struct SelectContext*, u32, s32);
+
 typedef struct {
-    GameState state;
-    char unk_A4[0x1DC];
+    /* 0x00 */ char* name;
+    /* 0x04 */ Select_LoadFunc loadFunc;
+    /* 0x08 */ s32 entranceIndex;
+} SceneSelectEntry; // size = 0xC
+
+typedef struct SelectContext {
+    /* 0x000 */ GameState state;
+    /* 0x0A8 */ View view;
+    /* 0x210 */ s32 count;
+    /* 0x214 */ SceneSelectEntry* scenes;
+    /* 0x218 */ s32 currentScene;
+    /* 0x21C */ s32 pageDownIndex; // Index of pageDownStops
+    /* 0x220 */ s32 pageDownStops[7];
+    /* 0x23C */ char unk_23C[0xC];
+    /* 0x248 */ s32 opt; // This name is the one the menu uses to show this value. It can be used to change in which entrance the player spawns in the selected scene
```

Comment:
So is this like the `entranceIndex`? or `currentEntrance`? I'm not sure if I like the name `opt`, what does it stand for?

Reply:
I think this is more like `entranceIndex`.
`opt` probably is a shorthand from "option".
It comes from the menu itself. It prints `OPT:` on screen.
![image](https://user-images.githubusercontent.com/7416381/134550371-372449c9-af4f-4051-8190-77203a34f269.png)

Reply:
```suggestion
    /* 0x248 */ s32 opt; // (Option): This name is the one the menu uses to show this value. It can be used to change in which entrance the player spawns in the selected scene
```
Nice, makes sense. Then I'd either add that to the comment or rename `opt` to `option` personally, but I'll leave it up to you